remoting/rico-remoting-server-spring-test/rico-remoting-server-spring-test.gradle

```diff
@@ -21,6 +21,12 @@ dependencies {
     implementation "org.testng:testng:$testngVersion"
     implementation "junit:junit:$junitVersion"
 
+    testCompile("org.junit.jupiter:junit-jupiter-api:5.7.0")
```

Comment:
Could you format the lines as the other dependencies (without the parentheses).
And also make the versions a variable in the `gradle.properties` file